changelog.d/4640.bugfix

```diff
@@ -0,0 +1,5 @@
+# [[Issue #4640]](https://github.com/vector-im/element-android/issues/4640) The notification badge jumps around
+
+## Changes :
+- change unread counter badge view in item_room to be in consistent place on the screen, to left of date/time.
```

Comment:
for the content of the entry we've been trying to keep them down to one liners, in this case we could have  

```
Fixing the room list notification badge jumping around
```

this content goes directly into the https://github.com/vector-im/element-android/blob/develop/CHANGES.md and release github release description

Reply:
It's updated. 
Thank you